src/protocols/ssh/settings.c

```diff
@@ -246,6 +247,15 @@ enum SSH_ARGS_IDX {
      * variable to be set.
      */
     IDX_LOCALE,
+     
+    /**
+     * The timezone that is passed from the client system to the
+     * remote server, or null if not specified.  If set, and allowed
```

Comment:
"... or null if not specified" is good to know, but not actually applicable to the connection parameter. This is an implementation detail which goes along with the internal representation within the settings struct. The documentation of the connection parameter should be from the perspective of a user considering providing a value for that parameter.

Reply:
Reworded, hopefully it's good.  Added reference to Wikipedia page.